assets/sass/v2/_okta-verify.scss

```diff
@@ -58,6 +58,11 @@
     text-decoration: none;
   }
 
+  .sameDevice-setup .download-ov-link {
+    color: #007cc0;
```

Comment:
Why is this link not also `color: $link-text-color`?

Reply:
good question, not really sure, I copied the style from orOnMobileLink.  I think I should change both of them to use `link-text-color`